csharp/BUILDING.md

```diff
@@ -28,8 +28,8 @@ A source build of Ice for .NET on Windows produces two sets of assemblies:
 In order to build Ice for .NET from source, you need:
 
 * A [supported version][3] of Visual Studio when building .NET Framework 4.5 Assemblies.
-* Visual Studio 2022 with [.NET 6.0 SDK][4] or [.NET 7.0 SDK][5] when building the .NET Standard 2.0 Assemblies.
-* Visual Studio 2022 with [.NET 6.0 SDK][4] and [.NET 7.0 SDK][5] when building the NuGet packages.
+* Visual Studio 2022 with [.NET 8.0 SDK][4] or [.NET 7.0 SDK][5] when building the .NET Standard 2.0 Assemblies.
+* Visual Studio 2022 with [.NET 8.0 SDK][4] and [.NET 7.0 SDK][5] when building the NuGet packages.
```

Comment:
I think we should just support .NET 8 and .NET 10 SDKs, the target frameworks are still netstandard 2.0 and .net45. Just build using a modern SDK